### PR TITLE
Use new wms addresses

### PR DESCRIPTION
**Pull request by oterral**
<jenkins>A test link will be added here if the deploy on int succeeded.</jenkins>

<jenkins>[Test link](https://mf-geoadmin3.int.bgdi.ch/fix_4232/index.html)</jenkins>

### CODE REVIEW

File: rc_prod [hunk 1]
```diff
@@ -5,12 +5,11 @@ export E2E_TARGETURL=https://map.geo.admin.ch
...
-export MAPPROXY_URL=//wmts{s}.geo.admin.ch
-export VECTORTILES_URL=//vectortiles{s}.geo.admin.ch
-export WMS_URL=//wms.geo.admin.ch
 export SHOP_URL=//shop.swisstopo.admin.ch
 export PUBLIC_URL=//public.geo.admin.ch
 export PRINT_URL=//print.geo.admin.ch
 export PROXY_URL=//service-proxy.prod.bgdi.ch
+export WMS_URL=//wms-bgdi-cdn-{s}.prod.bgdi.ch
 export WMTS_URL=//wmts{s}.geo.admin.ch
-export WMTS_TECH_URL=//wmts.
```
Comment (loicgasser) [inline rc_prod]: Is this wanted?

File: Makefile [hunk 2]
```diff
@@ -702,46 +687,17 @@ src/style/app.css: $(SRC_LESS_FILES)
...
-	    .build-artefacts/last-mapproxy-url \
-	    .build-artefacts/last-vectortiles-url \
-	    .build-artefacts/last-shop-url \
-	    .build-artefacts/last-wms-url \
-	    .build-artefacts/last-public-url \
-	    .build-artefacts/last-print-url \
-	    .build-artefacts/last-proxy-url \
-	    .build-artefacts/last-wmts-url \
-	    .build-artefacts/last-apache-base-path
+			${MAKO_LAST_VARIABLES}
```
Comment (loicgasser) [inline Makefile]: indent

File: Makefile [hunk 3]
```diff
@@ -702,46 +687,17 @@ src/style/app.css: $(SRC_LESS_FILES)
...
-	    .build-artefacts/last-alti-url \
-	    .build-artefacts/last-mapproxy-url \
-	    .build-artefacts/last-vectortiles-url \
-	    .build-artefacts/last-shop-url \
-	    .build-artefacts/last-wms-url \
-	    .build-artefacts/last-public-url \
-	    .build-artefacts/last-print-url \
-	    .build-artefacts/last-proxy-url \
-	    .build-artefacts/last-apache-base-path
+			${MAKO_LAST_VARIABLES}
```
Comment (loicgasser) [inline Makefile]: indent

File: rc_prod (hunk 1 above)
Comment (oterral) [reply rc_prod]: good catch

**Comment by oterral** [comment]
It works on it with which layer do you see that, I test with  thie [one](https://mf-geoadmin3.int.bgdi.ch/fix_4232/index.html?lang=fr&topic=luftbilder&bgLayer=ch.swisstopo.pixelkarte-grau&layers=ch.swisstopo.lubis-bildstreifen&layers_timestamp=&E=2637186.78&N=1190518.54&zoom=3&catalogNodes=1179,1180,1186,1184)

**Comment by oterral** [comment]
Ok I see the bug with aviation topic

File: Makefile (hunk 2 above)
Comment (oterral) [reply Makefile]: fixed

File: Makefile (hunk 3 above)
Comment (oterral) [reply Makefile]: fixed

**Comment by oterral** [comment]
Thx for the review @loicgasser it should work now.

**Comment by gjn** [comment]
Tested a little bit. Looks good to me.

**Pull request merged**